packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -49,24 +55,51 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
     @state()
     private _isModalOpen = false;
 
+    @state()
+    private locale: CookieBannerLocale = defaultLocale;
+
     @property({ type: Boolean })
     public hasPrimaryActionsOnly = defaultProps.hasPrimaryActionsOnly;
 
     @property({ type: Object })
     public defaultPreferences = defaultProps.defaultPreferences;
 
-    @property({ type: Object })
-    public locale:CookieBannerLocale = defaultProps.locale;
-
     @property({ type: String })
     public cookieStatementLink = defaultProps.cookieStatementLink;
 
     @property({ type: String })
     public cookieTechnologiesLink = defaultProps.cookieTechnologiesLink;
 
+    @property({ type: Tenant })
```

Comment:
Please can you use `String` here?
```suggestion
    @property({ type: String })
```